platform/core/src/actions/ActionCapability.js

```diff
@@ -54,6 +54,18 @@ define(
             this.domainObject = domainObject;
         }
 
+        function isEditable(domainObject){
+            return domainObject.getCapability('status').get('editing');
+        }
+
+        function hasEditableParent(domainObject){
+            return domainObject.hasCapability('context') &&
+                domainObject.getCapability('context').getPath().reduce(
```

Comment:
Using `some` instead of `reduce` would more clearly communicate the intent here and enable short-cutting behavior, and you could name your anonymous function to more clearly communicate what it does, e.g. 

```javascript
return domainObject.hasCapability('context') && 
    domainObject
        .getCapability('context')
        .getPath()
        .some(function isEditableAncestor(ancestor) {
            return ancestor.getCapability('status').get('editing');
        });
```